dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/ValidationReleasableFactoryImpl.java

```diff
@@ -145,15 +135,199 @@ protected void unsetManagedObjectService(ServiceReference<ManagedObjectService>
+        }
+
+        if (cvf != null) {
+            config.constraintValidatorFactory(cvf);
+        }
+    }
+
+//    private Set<ValueExtractorDescriptor> createValidationXmlValueExtractors(Configuration<?> config, ClassLoader appClassLoader) {
+//        BootstrapConfiguration bootstrapConfiguration = config.getBootstrapConfiguration();
+//        Set<String> valueExtractorFqcns = bootstrapConfiguration.getValueExtractorClassNames();
+//
+//        @SuppressWarnings("unchecked")
+//        Set<ValueExtractorDescriptor> valueExtractorDescriptors = valueExtractorFqcns.stream()
+//        .map( fqcn -> createInstance( (Class<? extends ValueExtractor<?>>) run( LoadClass.action( fqcn, null ) ) ) )
+//        .map( ve -> new ValueExtractorDescriptor( ve ) )
+//        .collect( Collectors.toSet() );
+//
+//        return valueExtractorDescriptors;
+//    }
+//
+//    @SuppressWarnings("rawtypes")
+//    private Set<ValueExtractorDescriptor> createServiceLoaderValueExtractors() {
+//        Set<ValueExtractorDescriptor> valueExtractorDescriptors = new HashSet<>();
+//
+//        List<ValueExtractor> valueExtractors = run( GetInstancesFromServiceLoader.action(
+//                                                                                         run( GetClassLoader.fromContext() ),
+//                                                                                         ValueExtractor.class
+//                        ) );
+//
+//        for ( ValueExtractor<?> valueExtractor : valueExtractors ) {
+//            valueExtractorDescriptors.add( new ValueExtractorDescriptor( injectInstance( valueExtractor ) ) );
+//        }
+//
+//        return valueExtractorDescriptors;
+//    }
+
+    private <T> ManagedObjectFactory<T> getManagedBeanManagedObjectFactory(Class<T> clazz) {
+        ModuleMetaData mmd = ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().getComponentMetaData().getModuleMetaData();
+        ManagedObjectService managedObjectService = managedObjectServiceRef.getService();
+        if (managedObjectService != null) {
+            try {
+                ManagedObjectFactory<T> factory = managedObjectService.createManagedObjectFactory(mmd, clazz, true);
+                if (factory.isManaged()) {
+                    return factory;
+                }
+            } catch (ManagedObjectException e) {
+                // ffdc
+            }
+        }
+        return null;
+    }
+
+    private <T> ManagedObject<?> createManagedObject(Object instance) {
+        ModuleMetaData mmd = ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().getComponentMetaData().getModuleMetaData();
+        ManagedObjectService managedObjectService = managedObjectServiceRef.getService();
+        if (managedObjectService != null) {
+            try {
+                return managedObjectService.createManagedObject(mmd, instance);
+            } catch (ManagedObjectException e) {
+                // ffdc
```

Comment:
can we log a debug message here in addition to logging FFDC?

Reply:
Added trace and will be opening follow-up issue.